HeterogeneousCore/AlpakaTest/test/testAlpakaModules_cfg.py

```diff
@@ -114,10 +118,13 @@
         mod = getattr(process, "alpaka"+name)
         mod.alpaka = cms.untracked.PSet(backend = cms.untracked.string(args.moduleBackend))
 if args.expectBackend == "cuda_async":
-    process.alpakaGlobalConsumer.expectSize = 20
-    process.alpakaStreamConsumer.expectSize = 25
-    process.alpakaStreamInstanceConsumer.expectSize = 36
-    process.alpakaStreamSynchronizingConsumer.expectSize = 20
+    def setExpect(m, size):
+        m.expectSize = size
+        m.expectBackend = "CudaAsync"
+    setExpect(process.alpakaGlobalConsumer, size=20)
+    setExpect(process.alpakaStreamConsumer, size=25)
+    setExpect(process.alpakaStreamInstanceConsumer, size=36)
+    setExpect(process.alpakaStreamSynchronizingConsumer, size=20)
```

Comment:
I prefer setting explicitly both values like it's done above, but I agree it's just a matter of taste.

Reply:
This test checks that the infrastructure of setting explicitly the per-backend values works, so I'd keep setting the checker-side expected size in this way.